packages/file-asset-apis/test/s3/s3-helpers-spec.ts

```diff
@@ -48,6 +48,14 @@ describe('S3 Helpers', () => {
             expect(nonExistent).toBeFalse();
         });
 
+        /// Need to disable this, Will only pass with minio RELEASE.2024-08-29T01-40-52Z and above
+        xit('should throw with an invalid bucket name', async () => {
+            const invalidBucket = 'NOT-valid-bucket';
+            expect(async () =>
+                await s3Helpers.doesBucketExist(client, { Bucket: invalidBucket })
```

Comment:
I think you should go ahead and test all of the ways that this function will throw or return true or false.